dbms/src/Core/materializeBlock.cpp

```diff
@@ -1,4 +1,4 @@
-#include <DataStreams/materializeBlock.h>
+#include "materializeBlock.h"
```

Comment:
Let's rename header to `MaterializedBlock.h`.

Reply:
Header name should match function name.
M* correct for class